### PR TITLE
Handle empty config load

### PR DESCRIPTION
**Pull request by NeonDaniel**
Catch and log an empty config file instead of handling in catch-all exception handling

### CODE REVIEW

File: ovos_config/models.py [hunk 1]
```diff
@@ -90,9 +90,12 @@ def load_local(self, path=None):
...
                         config = load_commented_json(path)
-                    for key in config:
-                        self.__setitem__(key, config[key])
-                    LOG.debug(f"Configuration {path} loaded")
+                    if config:
+                        for key in config:
+                            self.__setitem__(key, config[key])
+                        LOG.debug(f"Configuration {path} loaded")
+                    else:
+                        LOG.warning(f"Empty config found at: {path}")
```
Comment (JarbasAl) [inline ovos_config/models.py:98]: this should not be a warning, it is not an error or bad thing, can it be debug ?

Comment (NeonDaniel) [reply ovos_config/models.py:98]: I wasn't sure if there's ever a case where an empty config file makes sense, but I can reduce to a debug since the load message is also a debug

File: ovos_config/locations.py [hunk 2]
```diff
@@ -16,7 +16,7 @@
 from time import sleep
 import ovos_config.meta as _ovos_config
 from ovos_utils.xdg_utils import xdg_config_dirs, xdg_config_home, xdg_data_dirs, xdg_data_home, xdg_cache_home
-
+from ovos_utils.log import LOG
```
Comment (JarbasAl) [inline ovos_config/locations.py:19]: why was this import added? is it used somewhere below not flagged by the github diff?

Comment (NeonDaniel) [reply ovos_config/locations.py]: Nope, leftover from debugging
